cpp/oneapi/dal/algo/jaccard/backend/cpu/select_kernel.cpp

```diff
@@ -32,12 +32,12 @@ similarity_result backend_default<Float, Method, Graph>::operator()(
 }
 
 template <>
-dal::detail::pimpl<backend_base<undirected_adjacency_array<> &>>
-get_backend<float, method::by_default, undirected_adjacency_array<> &>(
+dal::detail::pimpl<backend_base<undirected_adjacency_array_graph<> &>>
```

Comment:
minor: would it make sense to use shorter alias for this long name in the implementation?